src/distributions/local/installer.ts

```diff
@@ -0,0 +1,78 @@
+import * as tc from '@actions/tool-cache';
+import * as core from '@actions/core';
+
+import fs from 'fs';
+import path from 'path';
+import semver from 'semver';
+
+import { JavaBase } from '../base-installer';
+import { JavaInstallerOptions, JavaDownloadRelease, JavaInstallerResults } from '../base-models';
+import { extractJdkFile } from '../../util';
+import { MACOS_JAVA_CONTENT_POSTFIX } from '../../constants';
+
+export class LocalDistribution extends JavaBase {
+  constructor(installerOptions: JavaInstallerOptions, private jdkFile?: string) {
+    super('jdkfile', installerOptions);
+  }
+
+  public async setupJava(): Promise<JavaInstallerResults> {
+    let foundJava = this.findInToolcache();
+
+    if (foundJava) {
+      core.info(`Resolved Java ${foundJava.version} from tool-cache`);
+    } else {
+      core.info(
+        `Java ${this.version.raw} is not found in tool-cache. Trying to unpack JDK file...`
```

Comment:
`Java ${this.version.raw} was not found in the tool-cache. Trying to ...`

Same thing line 34. `JDK file was not found...`